auto_rx/autorx/config.py

```diff
@@ -135,12 +135,16 @@ def read_auto_rx_config(filename, no_sdr_test=False):
 		# Debugging settings
 		'save_detection_audio' : False,
 		'save_decode_audio' : False,
-		'save_decode_iq' : False
+		'save_decode_iq' : False,
+		# URL for the Habitat DB Server.
+		# As of July 2018 we send via sondehub.org, which will allow us to eventually transition away
+		# from using the habhub.org tracker, and leave it for use by High-Altitude Balloon Hobbyists.
+		# For now, sondehub.org just acts as a proxy to habhub.org.
+		# This setting is not exposed to users as it's only used for unit/int testing
+		'habitat_url': "https://habitat.sondehub.org/"
 
 	}
 
-	sdr_settings = {}#'0':{'ppm':0, 'gain':-1, 'bias': False}}
```

Comment:
This doesn't appear to be used anywhere?

Reply:
Looks like it...